frontend/src/features/mapSideBar/SidebarContents/LandForm.tsx

```diff
@@ -113,11 +121,17 @@ const Form: React.FC<ILandForm> = ({
   setMovingPinNameSpace,
   handlePidChange,
   handlePinChange,
+  formikRef,
   isAdmin,
 }) => {
   // access the stepper to later split the form into segments
   const stepper = useFormStepper();
-  const formikProps = useFormikContext<IParcel>();
+  const formikProps = useFormikContext<ISteppedFormValues<IFormParcel>>();
+  useParcelLayerData({
+    formikRef,
+    nameSpace: 'data',
+    agencyId: +formikProps.values.data.agencyId,
```

Comment:
What is the `+` do for you here?

Reply:
in testing the value of agencyId was a string representation of an int in some situations.